mediaflow_proxy/extractors/vixcloud.py

```
@@ -39,31 +39,30 @@ async def version(self, site_url: str) -> str:
+            response = await self._make_request(url)
+        
         if response.status_code != 200:
             raise ExtractorError("Failed to extract URL components, Invalid Request")
         soup = BeautifulSoup(response.text, "lxml", parse_only=SoupStrainer("body"))
         if soup:
             script = soup.find("body").find("script").text
             token = re.search(r"'token':\s*'(\w+)'", script).group(1)
             expires = re.search(r"'expires':\s*'(\d+)'", script).group(1)
-            vixid = iframe.split("/embed/")[1].split("?")[0]
-            base_url = iframe.split("://")[1].split("/")[0]
+            canPlayFHD = re.search(r"window\.canPlayFHD\s*=\s*(\w+)", script).group(1)
```

Ruff (0.11.9)

Local variable `canPlayFHD` is assigned to but never used

Remove assignment to unused variable `canPlayFHD`

(F841)